### PR TITLE
fixes per sanak #2371

### PR DESCRIPTION
**Pull request by darkblue-b**
osgeolive trac ticket #2371

### CODE REVIEW

**Comment by sanak** [comment]
@darkblue-b Thanks for making PR !
I will try to build the branch locally tonight on my new Ubuntu 22.04 + VirtualBox environment.

File: bin/inchroot.sh [hunk 1]
```diff
@@ -92,6 +92,7 @@ adduser user --disabled-password --gecos user
 # TODO: Set the password for "user"
 mkdir -p /home/user/Desktop
 chown user:user /home/user/Desktop
+chmod 755 /home/user                 ## ticket #2371
```
Comment (kalxas) [inline bin/inchroot.sh:95]: I am not sure if home user folder permissions are related to this issue.
Does QGIS Server read anything from the user home folder?

**Comment by kalxas** [comment]
I am going to merge but will try a build without that chmod line above...

File: bin/inchroot.sh (hunk 1 above)
Comment (sanak) [reply bin/inchroot.sh:95]: @kalxas Well, now I am building `darkblue-b:qgis-serv7` image locally, so I can't launch another VM, but QGIS Server example reads National Earth 2 data from `/home/user/data/xxx` as symbolic link or something...
https://github.com/OSGeo/OSGeoLive/blob/master/app-data/qgis-server/themes.json#L80

**Comment by sanak** [comment]
> I am going to merge but will try a build without that chmod line above...

Okay, but without the chmod line, Natural Earth shape files can't be read from QGIS Server.

File: bin/inchroot.sh (hunk 1 above)
Comment (sanak) [reply bin/inchroot.sh:95]: Well, the following QGIS project file reads shape files from `/home/user/data/natural_earth2/*.shp`.
https://github.com/OSGeo/OSGeoLive/blob/master/app-data/qgis/QGIS-NaturalEarth-Example.qgz
 +- `/home/user/data/natural_earth2/ne_10m_lakes.shp`
 +- :

Comment (kalxas) [reply bin/inchroot.sh:95]: Thank you for the feedback, we should change the path to read the shapefiles outside of /home

Comment (sanak) [reply bin/inchroot.sh:95]: @kalxas Oh, yes. Thanks for confirmation!

From local environment trial, changing `natural_earth2` folder's owner (`sudo chown -R root:users /usr/local/share/data/natural_earth2`) seems to be also necessary.

I am preparing the another PR about this (from the latest `master` branch), so please wait for it until tomorrow.

**Pull request merged**